browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -313,7 +308,11 @@ bool ShouldBlockWalletWebUI(content::BrowserContext* browser_context,
   }
   auto* keyring_service =
       brave_wallet::KeyringServiceFactory::GetServiceForContext(profile);
-  return keyring_service && keyring_service->IsLockedSync();
+  // Support to unlock Wallet has been extended also through WebUI,
+  // so we block only when Wallet hasn't been created yet, as onboarding
+  // is offered only via native Andrioid UI.
+  return !keyring_service ||
```

Comment:
I believe changing this will allow for bypassing locking screens in some cases. I'm doing some further testing on this, but I think we need to be careful about this change here.

Reply:
This seems to be working fine, and after further testing I wasn't able to produce any sort of bypasses on screens